src/Routes/general.routes.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express")
+const generalRoute = express.Router()
+const generalController = require("../controllers/general.controller")
+
+/**
+ * @swagger
+ *  components:
+ *      schemas:
+ *          doctorSearch:
+ *             type: object
+ *             properties:
+ *                 doctorName:
+ *                     type: string
+ *                 doctorDepartment:
+ *                     type: string
+ *                 hospitalId:
+ *                     type: string
+ *                 hospitalName:
+ *                     type: string
+ */
+
+
+/**
+ * @swagger
+ * /general:
+ *  get:
+ *      summary: welcome to site
+ *      tags:
+ *          - general
+ *      description: welcome to site
+ *      responses:
+ *          200:
+ *              description: To test get method
+ */
+generalRoute.get('/',generalController.welcome )
+
+/**
+ * @swagger
+ * /general/hospitalList:
+ *  get:
+ *      summary: The get hospital data from database
+ *      tags:
+ *          - general  
+ *      description: displaying all hospital data from database
+ *      responses:
+ *          200:
+ *              description: successfully  displaying all product data from database
+ *              
+ */ 
+
+generalRoute.get('/hospitalList', generalController.hospitalList)
```

Comment:
Refer to this site for more API best practices and also it will make you understand how to divide routes https://www.partech.nl/nl/publicaties/2020/07/9-trending-best-practices-for-rest-api-development

Reply:
I have looked thought is and have implemented as much of the practices as posssible [(changes)](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/3/files#r1045540661)

Reply:
👍